dockers/ipu-pipelines-runner/Dockerfile

```
@@ -0,0 +1,33 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG PYTHON_VERSION=3.8
+
+FROM pytorchlightning/pytorch_lightning:base-ipu-py${PYTHON_VERSION}
+
+MAINTAINER PyTorchLightning <https://github.com/PyTorchLightning>
+
+RUN echo "ALL ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+WORKDIR /azp
+
+COPY ./dockers/ipu-pipelines-runner/start.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/start.sh
+
+COPY ./dockers/ipu-pipelines-runner/installdependencies.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/installdependencies.sh
+RUN /usr/local/bin/installdependencies.sh
```

```suggestion
COPY ./dockers/ipu-pipelines-runner/start.sh /usr/local/bin/
COPY ./dockers/ipu-pipelines-runner/installdependencies.sh /usr/local/bin/

RUN \
    chmod +x /usr/local/bin/start.sh && \
    chmod +x /usr/local/bin/installdependencies.sh && \
    /usr/local/bin/installdependencies.sh
```
less layer is better